src/google/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "google-extractor"
-version = "0.1.1"
+version = "0.1.2"
 description = ""
 authors = ["Brad Banister <brad@doublelinepartners.com>"]
```

Comment:
In other projects we generally have authorship as `Ed-Fi Alliance, LLC, and contributors`. Wonder if that would make sense here?

Reply:
Also, please add: `license = Apache-2.0`

Reply:
Sounds good.  Author was auto-generated.